documentation/2-options.md

```diff
@@ -290,6 +290,21 @@ console.log(data);
 //=> 'Hello, world!'
 ```
 
+Since Got 12, you can use spec-compliant [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData) objects as request body, such as [`formdata-node`](https://github.com/octet-stream/form-data) or [`formdata-polyfill`](https://github.com/jimmywarting/FormData):
+
+```js
+import got from 'got';
+import {FormData} from 'formdata-node'; // or 'formdata-polyfill/esm.min.js'
+
+const form = new FormData();
+
+form.set('greeting', 'Hello, world!');
+
+const {data} = await got.post('https://httpbin.org/post', {
+	body: form
+});
+```
```

Comment:
The docs here needs to be added to TS as doc comment too.

Reply:
You mean in Options class?

Reply:
Yes, where the other `body` option docs are.

Reply:
Done.